test/mysql.discover.test.js

```diff
@@ -271,4 +277,34 @@ describe('Discover and build models', function() {
       done();
     });
   });
+
+  describe('with treatCHAR1AsBoolean', function() {
+    var models;
+    before(function(done) {
+      db.discoverAndBuildModels('INVENTORY', {
+        owner: 'STRONGLOOP',
+        visited: {},
+        associations: true,
+        treatCHAR1AsBoolean: true,
+      }, function(err, models_) {
+        models = models_;
+        done(err);
+      });
+    });
+
+    it('should discover and build models', function() {
```

Comment:
@Amir-61 I used 'should' as all other tests in test/mysql.discover.test.js uses it("should...");